docs/install.rst

```diff
@@ -106,15 +113,42 @@ package, depending on how you wish to handle virtual environments:
 
    ``/path/to/jupyter/env`` is the path to your base conda environment, and depends on the options set during your initial installation. It's probably something like ``~/conda`` or ``~/miniconda``. See the `ipython docs <https://ipython.readthedocs.io/en/stable/install/kernel_install.html>`_ for more details.
 
+   With this setup, similar to option 2, you have a single jupyter install that can run kernels from any virtual environment. The main difference is that it can run kernels from *any* virtual environment (not just conda!) but that you have to run an additional line after installing ``ipykernel``  into the environment (``python -m ipykernel install ...``).
```

Comment:
Maybe mention another main difference: there is less stuff installed in base: no need to install `nb_conda_kernels` because you are instead pointing toward the base jupyter path (or whatever) from your virtual environment, so there is an even bigger savings than in option 2. 

This is really useful stuff, going above and beyond standard installation help.  :smile: